devel/imgui/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        ocornut imgui 1.83 v
+revision            0
+
+categories          devel graphics games
+platforms           darwin
+license             MIT
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+
+description         Bloat-free Graphical User interface for C++
+long_description    \
+    Dear ImGui is a bloat-free graphical user interface library for C++. \
+    It outputs optimized vertex buffers that you can render anytime in your \
+    3D-pipeline enabled application. It is fast, portable, renderer agnostic \
+    and self-contained (no external dependencies).\nDear ImGui is designed to \
+    enable fast iterations and to empower programmers to create content \
+    creation tools and visualization / debug tools (as opposed to UI for the \
+    average end-user). It favors simplicity and productivity toward this \
+    goal, and lacks certain features normally found in more high-level \
+    libraries.\nDear ImGui is particularly suited to integration in games \
+    engine (for tooling), real-time 3D applications, fullscreen applications, \
+    embedded applications, or any applications on consoles platforms where \
+    operating system features are non-standard.
+
+checksums           rmd160 ce2df5295f32903d58d9ef4b2378147ddc362fec \
+                    sha256 3de7af5078882edbebff452f2a9ee362ae05107896e14f6ab33621a3473eb95a \
+                    size 1448525
+
+post-extract {
+    # https://raw.githubusercontent.com/microsoft/vcpkg/master/ports/imgui/CMakeLists.txt
+    # https://raw.githubusercontent.com/microsoft/vcpkg/master/ports/imgui/imgui-config.cmake.in
+    copy ${filespath}/CMakeLists.txt ${worksrcpath}
+    copy ${filespath}/imgui-config.cmake.in ${worksrcpath}
+}
```

Comment:
why is upstream not providing these files if they're needed? Isn't it a bit unconventional that you need to grab installation/build files from a "random" repository?